plugins/database/src/db_plugin.cpp

```diff
@@ -618,27 +637,28 @@ irods::error _childIsValid(
     return SUCCESS();
 }
 
-irods::error _updateChildParent(
-    const std::string& _child_resc_id,
-    const std::string& _parent_resc_id,
-    const std::string& _parent_child_context ) {
+irods::error _updateChildParent(const std::string& _child_resc_id,
+                                const std::string& _parent_resc_id,
+                                const std::string& _parent_child_context)
+{
     irods::sql_logger logger( "_updateChildParent", logSQL );
 
+    const auto [secs, millis] = get_current_time();
```

Comment:
Consider different names for these...

`secs` -> `current_time_sec` (or `*_s` or `*_seconds` or...)
`millis` -> `current_time_msec` (or `*_ms` or `*_millis` or `*_milliseconds` or...)

Reply:
Done.